packages/components/src/ProgressBar/ProgressBar.tsx

```diff
@@ -19,15 +19,53 @@ interface ProgressBarProps {
    * @default base
    */
   readonly size?: keyof typeof sizes;
+
+  /**
+   * Set the variation to the stepped progress bar
+   * @default false
+   */
+  readonly stepped?: boolean;
```

Comment:
Do we want this to be boolean? Referencing @irmalcol comment [here](https://getjobber.slack.com/archives/C03BJHV3PMG/p1700666198355529?thread_ts=1698257310.303899&cid=C03BJHV3PMG), it sounds like we have a bit more flexibility with this property if we treat it as a string 

Would something like 

```
  @default progress
  variation?: progress | stepped; 
```

work here?

Reply:
Oops, this slipped my mind. Agree that having this as a string would be this more flexible for potential additions in the future! Changed in [`bce3e13`](https://github.com/GetJobber/atlantis/pull/1653/commits/bce3e13933b1f7fe6343fa71eba7647e6a575b83).